gmail/src/models/lead.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Represent a "crm.lead" record.
+ */
+
+export class Lead {
```

Comment:
We don't have as many models as in Odoo, so I think it's a bit overkill. And because the "Partner" can exists without "res.partner" (e.g. when we are not connected to a Odoo database)

Reply:
(and also to simplify the code as much as possible)